Source/Invoke-PackerBuild.ps1

```diff
@@ -6,8 +6,6 @@ param(
     [Parameter(Mandatory=$false,ParameterSetName="AWS EC2 AMI")]
     [Parameter(Mandatory=$true,ParameterSetName="Vagrant Hyper-V")]
     [string]$MockConnectionString=$null,
-    [Parameter(Mandatory=$false,ParameterSetName="AWS EC2 AMI")]
-    [string]$SourceAMI,
     [Parameter(Mandatory=$true,ParameterSetName="AWS EC2 AMI")]
     [string]$IAMInstanceProfile,
     [Parameter(Mandatory=$true,ParameterSetName="AWS EC2 AMI")]
```

Comment:
The region should also be available for the ParameterSetName "Vagrant Hyper-V".
Maybe make this optional, but add a default value through New-ISHBootstrapAWSReferences.ps1 ?